chart/values.yaml

```diff
@@ -58,6 +58,9 @@ config:
   awsRegion: ""
   awsSecretAccessKey: ""
   baaahThreadiness: "20"
+  bootstrapToken: "" # Randomly generated at runtime if not set
+  enableAuthentication: false
+  enableBootstrapUser: true
```

Comment:
These are the defaults for Obot itself, but I'm not sure if we want them to be the Helm defaults or not.

Reply:
Heard back from Craig. We want auth enabled on the chart by default, so I flipped that one.